tests/load_test/requirements.txt

```diff
@@ -0,0 +1,3 @@
+locustio
```

Comment:
Can you include package versions for these requirements?

Reply:
Great catch, will freeze